src/ng-select/ng-select.component.ts

```diff
@@ -113,6 +113,15 @@ export class NgSelectComponent implements OnInit, OnDestroy, OnChanges, AfterVie
     get single() {
         return !this.multiple;
     }
+    @Output('loadingChange') loadingChange: EventEmitter<Boolean> = new EventEmitter<Boolean>();
+    get loading(): boolean {
+        return this.isLoading;
+    }
+    @Input()
+    set loading(value: boolean) {
```

Comment:
This looks super cool. It would be great if you can add demo with use case. You can put it in demo app `reactive-forms.component.ts` or create your own showing custom loading.

Reply:
would like to have it there and may add a demo in the future but right now i can't